.github/workflows/_data-plane.yml

```
@@ -1,6 +1,7 @@
-name: Build Artifacts
+name: Data plane
```

Might help identify the workflow so it's less error-prone which one to click.

```suggestion
name: "Build artifacts: Data plane"
```